player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```diff
@@ -41,7 +41,7 @@ endPage.controller("endPageController", function($scope, $rootScope, $state,$ele
     };
 
     $scope.replayContent = function() {
-        EventBus.dispatch("event:openUserSwitchingModal", {'logGEEvent': $scope.pluginInstance._isAvailable});
+        EventBus.dispatch("event:openSunbirdUserSwitchingModal", {'logGEEvent': $scope.pluginInstance._isAvailable});
```

Comment:
Don't change event name. Check if we send config `enableUserSwitcher: false` it should disable default your switcher

Reply:
Why we are adding default user switcher for Sunbird, Yes, we can disable default user switcher by checking `enableUserSwitcher: false` from HTML side but if we are adding then `event:openUserSwitchingModal` event will always dispatch for default user switcher and event will log also. In the following way, we can avoid this
- Do not add default user plugin for an irrespective plugin (don't add default user switcher for sunbird endPage)
- `ng-if` check on HTML
- conditional check in `replayContent()` of endpage plugin
Please let me know what we can implement.
-cc @vinukumar-vs